src/jats/importer/jats-reference-parser.ts

```diff
@@ -125,59 +107,47 @@ export const jatsReferenceParser = {
       const year = getTrimmedTextContent(element, 'year')
 
       if (year) {
-        bibliographyItem.issued = buildBibliographicDate({
+        bibliographyItem.issued = {
```

Comment:
Can we continue using `buildBibliographicDate`?

Reply:
Sure.